srcpkgs/BAND/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'BAND'
+pkgname=BAND
+version=1.10.9
+revision=1
+archs="x86_64"
+hostmakedepends="binutils tar xz gzip"
+depends="gtk+3 libnotify nss libXScrnSaver libXtst xdg-utils at-spi2-core libuuid libappindicator libsecret"
+short_desc="Group Chat Client for Sports, School, Campus Groups, and more"
+maintainer="Adam Pschorr <adampschorr13@aol.com>"
+license="custom:Proprietary"
+homepage="https://band.us/home"
+distfiles="https://ssl.pstatic.net/cmstatic/desktop/v${version}/BAND-${version}-amd64.deb"
+checksum=88bbc33c1d831e51abad81cb43d4622382e3cb40de38f35d549effc979ea1f8c
+repository=nonfree
+restricted=yes
+do_extract() {
+	ar x ${XBPS_SRCDISTDIR}/BAND-${version}/BAND-${version}-amd64.deb
+	tar -xf data.tar.xz
+	gzip -d usr/share/doc/band/changelog.gz
+}
+do_install() {
+	vmkdir /opt/BAND
```

Comment:
because this is using `/opt`, xbps may remove it if no other package exists in `/opt`. Add something like [this](https://github.com/void-linux/void-packages/blob/master/srcpkgs/vivaldi/REMOVE) to ensure that doesn't happen.